<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CodeinfoWidget</class>
 <widget class="QWidget" name="CodeinfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>275</height>
   </rect>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QTreeWidget" name="lstCodeinfo">
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="itemsExpandable">
      <bool>false</bool>
     </property>
     <property name="allColumnsShowFocus">
      <bool>true</bool>
     </property>
     <property name="columnCount">
      <number>5</number>
     </property>
     <column>
      <property name="text">
       <string>File</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Line</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Col</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Code</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Message</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout">
     <item>
      <spacer name="spacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="cmbActions">
       <property name="editable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtCommand">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="placeholderText">
        <string>Enter command</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRun">
       <property name="text">
        <string>Run</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClipboard">
       <property name="text">
        <string>Clipboard</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnFile">
       <property name="text">
        <string>File ...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
